<?xml version="1.0"?>

<launch>
	<group ns="/sensor_01">
        <remap from="/ati_force_sensor/ati_force_torque" to="wrench" />
        <rosparam command="load" file="$(find teach_tool)/launch/force_sensor_01.param.yaml" />
        <node pkg="ati_force_sensor" name="ati_force_sensor_1" type="ati_force_sensor" output="screen" />
	</group>


	<group ns="/sensor_02">
        <remap from="/ati_force_sensor/ati_force_torque" to="wrench" />
        <rosparam command="load" file="$(find teach_tool)/launch/force_sensor_02.param.yaml" />
        <node pkg="ati_force_sensor" name="ati_force_sensor_2" type="ati_force_sensor" output="screen" />
	</group>

        <group ns="/sensor_03">
        <remap from="/ati_force_sensor/ati_force_torque" to="wrench" />
        <rosparam command="load" file="$(find teach_tool)/launch/force_sensor_03.param.yaml" />
        <node pkg="ati_force_sensor" name="ati_force_sensor_3" type="ati_force_sensor" output="screen" />
	</group>

</launch>
